test/dialect/test_sqlite.py

```diff
@@ -462,6 +463,21 @@ def test_boolean_default(self):
             [(False,), (True,)],
         )
 
+    @testing.provide_metadata
```

Comment:
OK, I see this imitates the style that was there, which I'd like to change, but since you faithfully imitated what was there I gues that's on me...

Reply:
Sorry, as you guest I just imitated test code from above as I haven't delve to much into the codebase.

Reply:
not your fault!